### PR TITLE
Monogame3.8

### PR DESCRIPTION
**Pull request by Uhha**
Hi!
May I suggest a migration to Monogame 3.8 and netcore 3.1?

### CODE REVIEW

File: Source/MonoGame.QuakeConsole.DesktopGL.csproj [hunk 1]
```diff
@@ -1,21 +1,10 @@
...
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>QuakeConsole</RootNamespace>
-    <AssemblyName>MonoGame.QuakeConsole</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <TargetFrameworkProfile />
-    <FileAlignment>512</FileAlignment>
-    <MonoGamePlatform>DesktopGL</MonoGamePlatform>
-    <MonoGameContentBuilderExe>
-    </MonoGameContentBuilderExe>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (discosultan) [inline Source/MonoGame.QuakeConsole.DesktopGL.csproj:5]: I'm thinking if it makes more sense to use `netstandard2.0` here. More portable and the project doesn't use any `netcoreapp3.1` features.

File: Source/MonoGame.QuakeConsole.WindowsDX.csproj [hunk 2]
```diff
@@ -1,120 +1,45 @@
...
-    </MonoGameContentBuilderExe>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+
+<PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
     <DebugSymbols>true</DebugSymbols>
+    <OutputPath>bin\Debug\Windows\</OutputPath>
+    <DefineConstants>TRACE;DEBUG;LINUX;WINDOWSDX</DefineConstants>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```
Comment (discosultan) [inline Source/MonoGame.QuakeConsole.WindowsDX.csproj:12]: Is this required? The project shouldn't have any unsafe code.

File: Interpreters/ManualInterpreter/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -32,5 +26,3 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
```
Comment (discosultan) [inline Interpreters/ManualInterpreter/Properties/AssemblyInfo.cs:28]: All the assemblyinfo files can be deleted. Afaik, in .NET Core, the same metadata is provided through .csproj instead.

File: Source/MonoGame.QuakeConsole.DesktopGL.csproj (hunk 1 above)
Comment (Uhha) [reply Source/MonoGame.QuakeConsole.DesktopGL.csproj]: I agree, changed to netstandard2.0

File: Interpreters/ManualInterpreter/Properties/AssemblyInfo.cs (hunk 3 above)
Comment (Uhha) [reply Interpreters/ManualInterpreter/Properties/AssemblyInfo.cs]: Removed

File: Source/MonoGame.QuakeConsole.WindowsDX.csproj (hunk 2 above)
Comment (Uhha) [reply Source/MonoGame.QuakeConsole.WindowsDX.csproj]: It's not required - removed the line

**Comment by discosultan** [comment]
@Uhha Thank you! Much appreciated.

**Pull request merged**